carmin-server/server/resources/executions.py

```diff
@@ -46,7 +58,8 @@ def post(self, model, user, db_session):
                 return error
 
             execution, error = get_execution_as_model(
-                user.username, new_execution.identifier, db_session)
+                user.username,
+                get_execution(new_execution.identifier, db_session))
```

Comment:
Comment (1) Get execution maybe should be done in a different statement given that it may return None.